### ISSUE TITLE
Feature request: customize 'cancelable' in dialogs

### ISSUE THREAD

**Issue by clement9527** [apache/cordova-plugin-dialogs#105]
To customize the dialog to be cancelable is supported in ReactNative: https://facebook.github.io/react-native/docs/alert. 

Is there a plan to support that in Cordova down the road?

**Comment by janpio** [comment]
We are always open to Pull Requests adding useful functionality.

**Comment by brodybits** [comment]
[navigator.notification.confirm](https://github.com/apache/cordova-plugin-dialogs#navigatornotificationconfirm) seems to support the same functionality in this plugin

I think this should be fixed by improving the documentation.

**Comment by bozzaj** [comment]
This is incorrect. navigator.notification.confirm does not support the same functionality. Specifically, setCancelable is ALWAYS set to true: [Notification.java](https://github.com/apache/cordova-plugin-dialogs/blob/master/src/android/Notification.java#L224)

**Comment by vitto32** [comment]
Can #134 be merged?
